hack/.spelling_ignorewords

```diff
@@ -0,0 +1,2 @@
+# misspell ignore the following corrections, comma separated: fooa,boob
+cyclinder
```

Comment:
bad rebase?

Reply:
sorry, I don't understand what you mean, Could you explain more about this?

Reply:
I'm wondering why do we need this change at all for this KEP?

Reply:
my name (cyclinder) did not pass the misspell check, so I made these changes, but I am not sure if I need to open a new PR for it, I put it on this pr for now. I can open another PR if it can be confirmed.

Reply:
see https://github.com/kubernetes/enhancements/issues/4674 for tracking